stable/airflow/templates/deployments-flower.yaml

```diff
@@ -54,6 +54,11 @@ spec:
       {{- if .Values.flower.tolerations }}
       tolerations:
 {{ toYaml .Values.flower.tolerations | indent 8 }}
+      {{- end }}
+      serviceAccountName: {{ template "airflow.serviceAccountName" . }}
```

Comment:
Can this be none ?

Reply:
the service account is attached to worker and scheduler already using the same flag, it would not introduce extra dependency, I believe it is initialized to string "airflow" by default, and can be configured in variable